app/models/Post.py

```diff
@@ -96,9 +96,10 @@ def clean(self):
                                               ['extra', 'smarty'])
         if self.images:
             for image in self.images:
-                self.html_content = self.html_content.replace(
-                    'src="' + image.filename + '"',
-                    'src="' + image.url() + '"')
+                if image.filename in self.html_content:
```

Comment:
This loop replaces 
```html
<img src="photo.jpg" />
```
with
```html
<img src="/admin/media/uploads/photo.jpg" />
```

The reason why I'm adding that line is actually super complicated... here goes:

In the post generation script, I create a BlogPost with a list of Images associated with it. Those images are not in use in the markdown content (see: https://github.com/adicu/adi-website/pull/187/files#diff-c5f641ef43dab675daa708b55940b2eaR171).  But, because here, in the `clean()` function (which is called whenever a BlogPost is saved) I originally called `image.url()` on every image associated with the blog post, and because `.url()` calls `url_for` to get the live url of the image, and that can't be called without the app running, things break.

By putting a guard around this call to `image.url()`, I ensure that only images which appear in the `htmt_content` on `BlogPost.save()` will have `url()` called on them.  That way, we can get around the `url_for` problem by simply not putting any images in the test blog posts.